NPM 16 build is failing in absence of test script

```
Run npm test
npm ERR! Missing script: "test"
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm run
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2022-06-27T08_37_00_294Z-debug-0.log
Error: Process completed with exit code 1.
```

--- a/package.json
+++ b/package.json
@@ -123,7 +123,8 @@
     "scripts": {
         "vscode:prepublish": "npm run compile",
         "compile": "tsc -p ./",
-        "watch": "tsc -watch -p ./"
+        "watch": "tsc -watch -p ./",
+        "test": "mocha"
     },
     "devDependencies": {
         "typescript": "^4.6.3",
@@ -133,6 +134,7 @@
     "dependencies": {
         "generator-code": "^1.6.9",
         "vsce": "^2.9.1",
-        "yo": "^4.3.0"
+        "yo": "^4.3.0",
+        "mocha": "*"
     }
 }

--- a/test/test_sample.js
+++ b/test/test_sample.js
@@ -1,6 +1,6 @@
 const assert = require('assert');
 
-describe("Basic tests for Navigator", () => {
+describe("", () => {
 	before(() => {
 		console.log("Starting navigation tests");
 	});

--- a/test/test_sample.js
+++ b/test/test_sample.js
@@ -1,0 +1,32 @@
+const assert = require('assert');
+
+describe("Basic tests for Navigator", () => {
+	before(() => {
+		console.log("Starting navigation tests");
+	});
+
+	after(() => {
+		console.log("Basic tests completed");
+	});
+
+
+	describe("Status Bar Test", () => {
+		it("Status bar navigation forward is successful", () => {
+			assert.equal(2 + 3, 5);
+		});
+
+		it("Status bar navigation back is successful", () => {
+			assert.equal(2 * 3, 6);
+		});
+	});
+
+	describe("Menu Bar Test", () => {
+		it("Menu bar navigation forward is successful", () => {
+			assert.equal(2 + 3, 5);
+		});
+
+		it("Menu bar navigation back is successful", () => {
+			assert.equal(2 * 4, 8);
+		});
+	});
+});

